specification/cognitiveservices/data-plane/Luis/v2.0/LUIS Programmatic APIs v2.0.swagger.json

```diff
@@ -0,0 +1,5225 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LUIS Programmatic APIs v2.0",
+    "version": "1.0"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.com",
+    "parameters": [
+      {
+        "name": "AzureRegion",
+        "description": "Supported Azure regions for Cognitive Services endpoints",
+        "x-ms-parameter-location": "client",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true,
+        "x-nullable": "false",
+        "x-ms-enum": {
+          "name": "AzureRegions",
+          "modelAsString": false
+        },
+        "enum": [
+          "westus",
+          "westeurope"
+        ]
+      }
+    ]
+  },
+  "basePath": "/luis/api/v2.0",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    },
+    "apiKeyQuery": {
+      "type": "apiKey",
+      "name": "subscription-key",
+      "in": "query"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    },
+    {
+      "apiKeyQuery": []
+    }
+  ],
+  "paths": {
+    "/apps/{appId}/versions/{versionId}/patterns": {
+      "post": {
+        "description": "Creates a new pattern feature.",
+        "operationId": "features - Create pattern feature",
```

Comment:
operationID should be of a form similar to https://github.com/Azure/azure-rest-api-specs/blob/current/specification/cognitiveservices/data-plane/Face/v1.0/Face.json#L31 
The string before the underscore "_" groups the operations in a class named that way (in code generation)
Something like "Features_CreatePattern"
The name is used for the generated method name.